website/docs/explorer/querying.mdx

```diff
@@ -63,45 +63,34 @@ And represented in the UI like
 
 ![explorer](imgs/explorer-query-overview.png)
 
-## Querying
+## Query Syntax
 
-Based on the previous schema you have two main modes to discover or filter the data
+The Explorer query system uses the[Bleve](https://blevesearch.com) search indexer and the [query language](https://blevesearch.com/docs/Query-String-Query/) that accompanies it.
 
-1. Filter by key and value: when you know the pair <key,value> that you want to filter from.
-2. Filter by value: when you want to filter across keys or does not know the key
+A brief introduction to the query syntax:
 
-:::info Matching
-
-For `key:value` filter the results would be those results where `key=value` exactly.
-:::
-
-
-### Filter by key and value
-
-You search with the format `key:value` indicating that you want to filter the resources matching the key and value.
-A couple of examples could be:
-
-- `kind:HelmRelease` to filter all helm releases.
-- `status:Failed` to find all failing resources.
-
-### Filter by value
-You search with a single term `value` indicating that you want to filter across the supported keys by values.
-The value would be search cross keys: `name`, `namespace` and `cluster` as synthatic sugar for `name:value` OR `namespace:value` OR `cluster:value`.
-
-An example cold be using `podinfo` that would translate into `name:podinfo` OR `namespace:podinfo` OR `cluster:podinfo`.
-
-### Operators
+Simple string matching query will match on all fields:
+```
+podinfo
+```
 
-#### AND
+To add field scoping to a query, use a `:` between key and value:
+```
+kind:Kustomization
+```
 
-You could refine filtered results by using AND semantics adding different filters.
+To add required fields, use the `+` operator. Use the `-` to exclude certain fields. This will return all kustomizations NOT on the management cluster:
```

Comment:
```suggestion
To make a term mandatory, put `+` in front. Use `-` in front of a term to exclude matches from the result. This will return all kustomizations NOT on the management cluster:
```

This is my thinking: it's the term (field and value, or value in any field, to be matched) that's required or excluded.

Reply:
No longer relevant after moving to faceted search